src/gwt/src/org/rstudio/studio/client/common/rstudioapi/DialogHtmlSanitizer.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * DialogHtmlSanitizer.java
+ *
+ * Copyright (C) 2009-16 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+package org.rstudio.studio.client.common.rstudioapi;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.google.gwt.safehtml.shared.HtmlSanitizer;
+import com.google.gwt.safehtml.shared.SafeHtml;
+import com.google.gwt.safehtml.shared.SafeHtmlUtils;
+
+public final class DialogHtmlSanitizer implements HtmlSanitizer {
+   private static final DialogHtmlSanitizer INSTANCE = new DialogHtmlSanitizer();
+
+   private static final Set<String> TAG_WHITELIST = new HashSet<String>(
+      Arrays.asList(
+         "p", "em", "strong", "b", "i"
+      )
+   );
+
+   public static DialogHtmlSanitizer getInstance() {
```

Comment:
Looks like we don't use this or the `INSTANCE` anywhere -- since this class has no state can we just use all-static methods and remove the instance variable? (this is the pattern we use for most helper classes)